### ISSUE TITLE
Mono3D Video Loss Function

### ISSUE THREAD

**Issue by mcsuy1998** [wbhu/Mono3D#4]
You mentioned that you use a RNN version for stereo video input. However, how is the loss calculated. The formula provided is lambda_1 * MonoLoss + lamba_2* InverseLoss + StereoLoss. Are the losses added per time-step? Then loss.backward() is performed at the end of the batch?

**Comment by mcsuy1998** [comment]
Another way I see is that the loss is calculated per timestep so at time t and then loss.backward() is done at that time frame.

**Comment by mcsuy1998** [comment]
Actually, we can also concatenate the output per timeframe and compute the loss at the end of the batch. Is this how loss should be calculated?

**Comment by wbhu** [comment]
Hi @mcsuy1998 ,

Total loss formulation is shown in Eq. 3. We compute the total loss based on a batch of sequence, and then backward the loss.

**Comment by mcsuy1998** [comment]
I see. I wanted to ask how to train the video model. I know you initialized the video model with a pretrained image model. Is the learning rate of the video model the same as the image model? You also mentioned that video model uses extra 1x1 convolutions weights. Did you use a different learning rate for these additional weights?

**Comment by wbhu** [comment]
1. the learning rate was initially set to 0.0001 and further reduced by a factor of 3.33 when the loss plateaus (known as ReduceLRonPlateau in PyTorch)
2. We didn't adopt such a strategy. But I think it is worth a try to set a lower learning rate for the additional weights.
